packages/arb-token-bridge-ui/src/components/TransferPanel/TokenSearch.tsx

```diff
@@ -272,9 +292,17 @@ function TokensPanel({
           token = ARB_SEPOLIA_NATIVE_USDC_TOKEN
         }
 
+        if (isTokenArbitrumOneUSDCe(address) && isDepositMode && isOrbitChain) {
+          // hide USDC.e if depositing to an Orbit chain
+          return false
+        }
+
         // If the token on the list is used as a custom fee token, we remove the duplicate
-        if (nativeCurrency.isCustom && address !== NATIVE_CURRENCY_IDENTIFIER) {
-          return address.toLowerCase() !== nativeCurrency.address
+        if (
```

Comment:
bugfix: if the native currency is custom, this would always return. This always returned true for anything else than the native currency, and would display all bridged tokens - even if the balance is 0.